aunt-leahs-app/src/pages/volunteerPage/volunteerPage.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import '../../App.css';
+import SelectBox from '../../components/select-box/select-box.jsx'
+import CustomButton from '../../components/customButton/customButton.jsx'
+import Head from '../../components/header/header.jsx'
+
+const VolunteerPage = () => (
+	<div className="App">
+		<Head page="Volunteer Login" />
+		<div style={{ margin: 'auto', width: '200px' }}>
```

Comment:
Put css in a css file and import it. Don't write in-line css.

Reply:
Not too sure how to import css styling